cypress/e2e/news/createReadDeleteNewsTimeOptions.feature

```diff
@@ -0,0 +1,54 @@
+@stable_test
```

Comment:
feture file name should include term school -> createReadDeleteSchoolNews.feature